src/components/ContactUs/ContactUs.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const ContactUs2 = () => {
+  return (
+    <div className="w-72 h-auto py-8">
+      <form className="bg-white  rounded-2xl px-8 pt-6 pb-8 mb-4 border-2 border-orange-500">
+        <div className="mb-4">
+          <h1 className="block text-orange-500  text-lg font-medium mb-2">
+            Contact Us!
+          </h1>
+        </div>
+        <div className="mb-4">
+          <input
```

Comment:
please surround all `<input />` fields in a `<form />` field and make them `required` so later we can validate them in the `onSubmit` call